fetch.bs

```diff
@@ -4426,8 +4426,30 @@ steps:
   <ol>
    <li><p>Let <var>transformStream</var> be a new {{TransformStream}}.
 
-   <li><p>Let <var>identityTransformAlgorithm</var> be an algorithm which, given <var>chunk</var>,
-   <a for=TransformStream lt=enqueue>enqueues</a> <var>chunk</var> in <var>transformStream</var>.
+   <li>
+    <p>Let <var>identityTransformAlgorithm</var> be the following steps given
+    <a>byte sequence</a> <var>chunk</var>:
+
+    <ol>
+     <li><p><a for=TransformStream lt=enqueue>Enqueue</a> <var>chunk</var> in
+     <var>transformStream</var>.
+
+     <li>
+      <p>If <var>response</var>'s <a for=response>URL</a> is null, then:
```

Comment:
How does this work? This runs after this step:

> If internalResponse’s [URL list](https://fetch.spec.whatwg.org/#concept-response-url-list) [is empty](https://infra.spec.whatwg.org/#list-is-empty), then set it to a [clone](https://infra.spec.whatwg.org/#list-clone) of request’s [URL list](https://fetch.spec.whatwg.org/#concept-request-url-list).

(And note that there are other responses that don't have a URL too.)